specs/electra/beacon-chain.md

```diff
@@ -1341,24 +1340,18 @@ def apply_deposit(state: BeaconState,
         # Verify the deposit signature (proof of possession) which is not checked by the deposit contract
         if is_valid_deposit_signature(pubkey, withdrawal_credentials, amount, signature):
```

Comment:
How about this?
```python
        if not is_valid_deposit_signature(pubkey, withdrawal_credentials, amount, signature):
            return

        add_validator_to_registry(state, pubkey, withdrawal_credentials, Gwei(0))  # [Modified in Electra:EIP7251]
```

Reply:
Yes, good suggestion. I believe this is easier to read.

Reply:
I've had a change of opinion. I think the "if not is_valid" doesn't read very well.